README.md

```diff
@@ -1,2 +1,3 @@
 # Northcoders News API
 
+The environment variables have been included in the .gitignore file, so will need to be recreated in order to successfully connect. 
```

Comment:
what should be included in these? what should they be called? is there an example file you can point towards here?